docs/photoniq/vwrs/configure-waiting-rooms.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 20
+title: Configuring Waiting Rooms
+---
+
+This page explains what can be configured in each waiting room using API endpoint properties. For more information about how to create a waiting room, refer to [Configure a Domain](configure-domain.md).
+
+## Entering the Waiting Room
+
+- **Rate Limit:** Set the rate (maximum users per second) at which traffic should be diverted to the waiting room. (`rate_limit` property)
+- **Waiting Room Path:** The URL that the waiting room needs to sit in front of. (`domain_url` property)
+- **Queue Type:** Defines how requests should be removed from the waiting room, such as priority or FIFO (first in, first out). (`queue_type` property)
```

Comment:
There is a slight problem here.  queue_type can be set as one of these three:
https://www.macrometa.com/docs/photoniq/vwrs/queue-types
But technically, only FIFO only works.  (I wish we would remove this until we have more support, but I always seem to get pushback when I suggest that.)

Reply:
Once FIFO is configured, you can add priority the ability to add priority queues.
https://www.macrometa.com/docs/photoniq/vwrs/configure-vwrs-edgeworker
Look under `Priority`.

I would like to pull this `Priority` config into it's own page because we should discuss how it works etc.  Will post a PR when I have something.

Reply:
Okay, so what change should I make to this text?

Reply:
```suggestion
- **Queue Type:** Defines how requests should be removed from the waiting room, such as FIFO (first in, first out). (`queue_type` property)
```